sdf/1.11/mesh_shape.sdf

```diff
@@ -1,5 +1,12 @@
 <element name="mesh" required="0">
   <description>Mesh shape</description>
+
+  <attribute name="simplification" type="string" default="" required="0">
+    <description>
+      Set whether to simplify the mesh using one of the specified methods. Values include: "convex_hull", "convex_decomposition". Default value is an empty string which means no mesh simplification.
```

Comment:
Could it make sense to add a sentence, a link or a reference to what these methods mean?

Reply:
expanded this to include description for the values. [64d7c97](https://github.com/gazebosim/sdformat/pull/1380/commits/64d7c97607c953202b8a771a7c7b3a2e67873774)